doc/source/guide/tour.rst

```diff
@@ -142,6 +142,32 @@ available. ::
 These constants are provided as a convenience so that everyone is using the same 
 (accepted values). More will be added over time.
 
+Quantities and Units
+--------------------
+
+Many capabilities in SunPy make use of physical quantities that are specified
+in units. SunPy uses `astropy's units and quantities code <http://docs.astropy.org/en/stable/units/index.html>`__ to
+implement this functionality. For example, the solar radius above is a physical quantity
+that can be expressed in length units.  In the example above ::
+
+    >>> from sunpy.sun import constants as con
+    >>> con.radius
+    <Constant name=u'Solar radius' value=695508000.0 error=26000.0 units='m' reference=u"Allen's Astrophysical Quantities 4th Ed.">
+
+shows the solar radius in units of meters.  It is simple to express the same physical quantity in different units::
+
+    >>> con.radius.to('km')
+    <Quantity 695508.0 km>
+
+To get the numerical value of the solar radius in kilometers - without the unit information - use ::
+
+    >>> con.radius.to('km').value
+    695508.0
+
+Quantities and units are simple and powerful tools for keeping track of the units you're working in, and make it
+easy to convert the same physical quantity into different units.  SunPy's adoption of quantities and units is
+described `here <https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0003.md>`__.
+
```

Comment:
Perhaps a link to the astropy Quantity tutorial would suffice?
http://www.astropy.org/astropy-tutorials/Quantities.html

When it comes to describing astropy code in SunPy, I'd prefer to make use
of what astropy provides as opposed to generating new content, as the
astropy content will be up-to-date and written by people who have developed
that code.

On Wed, Jan 7, 2015 at 5:07 PM, Stuart Mumford <notifications@github.com>
wrote:

> In doc/source/guide/tour.rst
> <https://github.com/sunpy/sunpy/pull/1253#discussion-diff-22620385>:
>
> > +    <Constant name=u'Solar radius' value=695508000.0 error=26000.0 units='m' reference=u"Allen's Astrophysical Quantities 4th Ed.">
> > +
> > +shows the solar radius in units of meters.  It is simple to express the same physical quantity in different units::
> > +
> > +    >>> con.radius.to('km')
> > +    <Quantity 695508.0 km>
> > +
> > +To get the numerical value of the solar radius in kilometers - without the unit information - use ::
> > +
> > +    >>> con.radius.to('km').value
> > +    695508.0
> > +
> > +Quantities and units are simple and powerful tools for keeping track of the units you're working in, and make it
> > +easy to convert the same physical quantity into different units.  SunPy's adoption of quantities and units is
> > +described `here <https://github.com/sunpy/sunpy-SEP/blob/master/SEP-0003.md>`__.
> > +
>
> I would show some Quantity arithmetic here (and not from a constant as
> they are a little different) as well as calling a SunPy function and doing
> something with the return value, i.e. showing a link between the input unit
> and the output.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1253/files#r22620385>.
>